src/index.js

```diff
@@ -38,21 +49,22 @@ export default async ({ to, subject, html }) => {
         SES: { ses: sesClient, aws }
       })
     }
-
     const info = await transporter.sendMail({
       from: process.env.FROM_EMAIL_ADDRESS,
       to,
       subject,
       html,
       text: createTextVersion(html)
-    }).then(res => res)
+    }).then(res => res).catch(error => {
+      throw new InternalServerError(`Email sending error: ${error.message}`)
+    })
     return {
       status: 200,
       result: {
         info
       }
     }
   } catch (err) {
-    return new ValidationError('Check your sending to field')
+    return err
```

Comment:
Here you can put an if that checks if the type of the err is ValidationError or InternalServerError. If so, you can simply rethrow them. If not, here can come the 'email sending error'... (But that will only work if you remove the .then and .catch above.

Also, why do we need the .then(res => res) ?